### PR TITLE
Make sure to track who created funds and repos

### PR DESCRIPTION
**Pull request by bryanjos**
#### Description:
This change slightly backtracks on having many to many on users/funds and users/repos. That still exists, but now we capture who created funds and repos in incentive. Previous change was to begin introducing organizations, which can still happen. This does not prevent that at all

### CODE REVIEW

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/199).

File: lib/incentivize/funds/funds.ex [hunk 1]
```diff
@@ -20,6 +21,8 @@ defmodule Incentivize.Funds do
       |> Repo.update()
     end)
     |> Multi.run(:user_funds, fn %{update_fund: fund} ->
+      user = Users.get_user(fund.created_by_id)
```
Comment (chas-mcmahon) [inline lib/incentivize/funds/funds.ex]: just pass `fund.created_by_id`?

File: lib/incentivize/repos/repositories.ex [hunk 2]
```diff
@@ -27,10 +27,12 @@ defmodule Incentivize.Repositories do
     |> Repo.all()
   end
 
-  def create_repository(params, %User{} = user) do
+  def create_repository(params) do
     Multi.new()
     |> Multi.insert(:repository, Repository.create_changeset(%Repository{}, params))
     |> Multi.run(:user_repositories, fn %{repository: repo} ->
+      user = Users.get_user(repo.created_by_id)
```
[the remark this answers is not in the document]
Comment (bryanjos) [reply lib/incentivize/repos/repositories.ex]: Good catch!

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/199).

**Pull request merged**